packages/desktop-client/src/components/accounts/MobileAccounts.js

```diff
@@ -0,0 +1,358 @@
+
+    // If there are no accounts, show a helpful message
+    if (accounts.length === 0) {
+      return <EmptyMessage onAdd={onAddAccount} />;
+    }
+
+    const accountContent = (
+      <View style={{ overflowY: 'auto' }}>
+        <View
+          style={{
+            alignItems: 'flex-end',
+            backgroundColor: colors.b2,
+            color: 'white',
+            flexDirection: 'row',
+            flex: '1 0 auto',
+            fontSize: 18,
+            fontWeight: 500,
+            height: 50,
+            justifyContent: 'center',
+            overflowY: 'auto',
+            paddingBottom: 10
+          }}
+        >
+          Accounts
+        </View>
+        <View
+          style={{
+            backgroundColor: colors.n10,
+            overflowY: 'auto',
+            padding: 10
+          }}
+        >
+          <AccountHeader name="Budgeted" amount={getOnBudgetBalance()} />
+          {budgetedAccounts.map((acct, idx) => (
+            <AccountCard
+              account={acct}
+              key={acct.id}
+              updated={updatedAccounts.includes(acct.id)}
+              getBalanceQuery={getBalanceQuery}
+              onSelect={onSelectAccount}
+            />
+          ))}
+
+          <AccountHeader name="Off budget" amount={getOffBudgetBalance()} />
+          {offbudgetAccounts.map((acct, idx) => (
+            <AccountCard
+              account={acct}
+              key={acct.id}
+              updated={updatedAccounts.includes(acct.id)}
+              getBalanceQuery={getBalanceQuery}
+              onSelect={onSelectAccount}
+            />
+          ))}
+
+          {/*<Label
+          title="RECENT TRANSACTIONS"
+          style={{
+            textAlign: 'center',
+            marginTop: 50,
+            marginBottom: 20,
+            marginLeft: 10
+          }}
+          />*/}
+        </View>
+      </View>
+    );
+
+    return (
+      <View style={{ flex: 1 }}>
+        {/* <TransactionList
+          transactions={transactions}
+          categories={categories}
+          isNew={this.isNewTransaction}
+          scrollProps={{
+            ListHeaderComponent: accountContent
+          }}
+          // refreshControl={refreshControl}
+          onSelect={onSelectTransaction}
+        /> */}
+        {accountContent}
+      </View>
+    );
+  }
+}
+
+function Accounts(props) {
+  const transactions = useState({});
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const getAccounts = async () => {
+      if (props.categories.length === 0) {
+        await props.getCategories();
+      }
+
+      props.getAccounts();
+    };
+
+    getAccounts();
+  }, []);
+
+  // const sync = async () => {
+  //   await props.syncAndDownload();
+  // };
+
+  const onSelectAccount = id => {
+    navigate(`/accounts/${id}`);
+  };
+
+  const onSelectTransaction = transaction => {
+    navigate(`/transaction/${transaction}`);
+    // props.navigation.navigate('Transaction', { transaction });
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏can be removed
```suggestion
```